fluentd-daemonset/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM fluent/fluentd:v1.2.1-debian-onbuild
+MAINTAINER Petr Kotas <petr.kotas@gmail.com>
+
+RUN buildDeps="sudo make gcc g++ libc-dev ruby-dev" \
+ && apt-get update \
+ && apt-get install -y --no-install-recommends $buildDeps \
+ && sudo gem install \
+        fluent-plugin-secure-forward \
```

Comment:
Secure forward should work out of the box, no need for installing it.

Reply:
@petrkotas see here: https://github.com/tagomoris/fluent-plugin-secure-forward on why it is not needed.